bin/k8s-apply

```diff
@@ -3,35 +3,39 @@
 . k8s-read-config
 
 echo "Updating ConfigMaps"
-for i in "${CONFIGMAP_FILES[@]}"
+for index in "${!CONFIGMAP_FILES[@]}"
```

Comment:
This was mainly an attempt at making looping structure consistent across scripts. I saw this structure through k8s-deploy on `master` and wondered why the two styles were used to seemingly similar effect. Not attached to the modification if you'd like to revert it.